### PR TITLE
Trim input parameters

### PR DESCRIPTION
**Pull request by carabasdaniel**
Trim input parameters

### CODE REVIEW

**Comment by carabasdaniel** [comment]
verify this please

File: src/CloudFoundry.Build.Tasks/src/BaseTask.cs [hunk 1]
```diff
@@ -74,6 +78,8 @@ internal CloudFoundryClient InitClient()
                     }
                 }
 
+                this.CFPassword = this.CFPassword.Trim();
```
Comment (viovanov) [inline src/CloudFoundry.Build.Tasks/src/BaseTask.cs]: I am not sure about this one. Can I have a space in my password in Cloud Foundry?

**Comment by carabasdaniel** [comment]
verify this please

**Comment by mihaibuzgau** [comment]
:+1:

**Pull request merged**